### ISSUE TITLE
VS Code: Able to see "Azure Function Operations" connector even after disabling/skipping the use of connectors from Azure

### ISSUE THREAD

**Issue by vgouth** [Azure/LogicAppsUX#2836]
### Describe the Bug with repro steps

1.Create a new workflow app using VS Code and **DO NOT enable** Azure connectors to be used
2.Create a workflow (stateful or stateless)
3.Add a HTTP trigger to the workflow and save the workflow
4.Add an action after the trigger
5.Take note of the actions available to you, you should not be able to see "Azure Function Operations" connector.

**Expected:** Should **not** be able to see "Azure Function Operations" connector.
**Actual:** Able to see "Azure Function Operations" connector even after disabling/skipping the use of connectors from Azure

### What type of Logic App Is this happening in?

Standard (VSCode)

### Are you using Preview Designer or GA Designer

Preview

### Did you refer to the TSG before filing this issue? https://aka.ms/lauxtsg

No

### Workflow JSON

```json
{
    "definition": {
        "$schema": "https://schema.management.azure.com/providers/Microsoft.Logic/schemas/2016-06-01/workflowdefinition.json#",
        "actions": {},
        "contentVersion": "1.0.0.0",
        "outputs": {},
        "triggers": {
            "When_a_HTTP_request_is_received": {
                "type": "Request",
                "kind": "Http"
            }
        },
        "parameters": {}
    },
    "kind": "Stateful"
}
```

### Screenshots or Videos

![image](https://github.com/Azure/LogicAppsUX/assets/113517384/948f17f0-f666-4854-8360-1979db0c01f0)
![image](https://github.com/Azure/LogicAppsUX/assets/113517384/9c7b3b76-1cbb-4c74-a02c-53cbb79e056f)
![image](https://github.com/Azure/LogicAppsUX/assets/113517384/f178c27e-b707-4af4-b000-afefc2dbcd30)

### Browser

VS Code

### Additional context

**Artifacts details:**

1.VS Code extension: [vscode-azurelogicapps-2.28.3.vsix](https://microsoft.sharepoint.com/:u:/t/LogicApps/ERQvUcsm27ZDqAk8h0IZFCQBnKEudkPL03CVtbYt1v_VMA?e=t0HVLO&xsdata=MDV8MDF8di1nb3Z1YmlsaXNoQG1pY3Jvc29mdC5jb218ZmM2NDU5N2YxZDQxNGVlYmVlYTcwOGRiNzM4MjM4YmN8NzJmOTg4YmY4NmYxNDFhZjkxYWIyZDdjZDAxMWRiNDd8MXwwfDYzODIzMDc3NTQ4NDkwNTEwOHxVbmtub3dufFRXRnBiR1pzYjNkOGV5SldJam9pTUM0d0xqQXdNREFpTENKUUlqb2lWMmx1TXpJaUxDSkJUaUk2SWsxaGFXd2lMQ0pYVkNJNk1uMD18MzAwMHx8fA%3d%3d&sdata=TDhwSU14VSsyOVNxNkErMVIyY0QyZkd4TGZwQTIzYktQYWxxM2ZHNnNTOD0%3d)
2.Nuget package:
[https://artprodeussu2.artifacts.visualstudio.com/A41bf5486-7392-4b7a-a7e3-a735c767e3b3/b32aa71e-8ed2-41b2-9d77-5bc261222004/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL21zYXp1cmUvcHJvamVjdElkL2IzMmFhNzFlLThlZDItNDFiMi05ZDc3LTViYzI2MTIyMjAwNC9idWlsZElkLzc1MzA5MDMwL2FydGlmYWN0TmFtZS9kcm9wX2J1aWxkX0J1aWxkQXJ0aWZhY3RzX1dpbmRvd3M1/content?format=file&subPath=%2FMicrosoft.Azure.Workflows.WebJobs.Extension.1.28.2.nupkg](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fartprodeussu2.artifacts.visualstudio.com%2FA41bf5486-7392-4b7a-a7e3-a735c767e3b3%2Fb32aa71e-8ed2-41b2-9d77-5bc261222004%2F_apis%2Fartifact%2FcGlwZWxpbmVhcnRpZmFjdDovL21zYXp1cmUvcHJvamVjdElkL2IzMmFhNzFlLThlZDItNDFiMi05ZDc3LTViYzI2MTIyMjAwNC9idWlsZElkLzc1MzA5MDMwL2FydGlmYWN0TmFtZS9kcm9wX2J1aWxkX0J1aWxkQXJ0aWZhY3RzX1dpbmRvd3M1%2Fcontent%3Fformat%3Dfile%26subPath%3D%252FMicrosoft.Azure.Workflows.WebJobs.Extension.1.28.2.nupkg&data=05%7C01%7Cv-govubilish%40microsoft.com%7Cfc64597f1d414eebeea708db738238bc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638230775484905108%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=g10UtlCyrfuCJ3xEYvwnFQPrl%2Bm6XQ2kcVQMsojBn4U%3D&reserved=0)
3.Extension Bundle (CDN URL): [https://cdnforlogicappsv2.blob.core.windows.net/logicapps-vendortesting](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcdnforlogicappsv2.blob.core.windows.net%2Flogicapps-vendortesting&data=05%7C01%7Cv-govubilish%40microsoft.com%7Ce347346c9dcc4a4cf80808db6b188892%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638221525391045919%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=%2BMcHLJsKeduA0dQVQ2CgOL5GzXjsVgOPWUrnxLtW%2Fnw%3D&reserved=0)
4.Extension Bundle (Direct Download): [https://cdnforlogicappsv2.blob.core.windows.net/logicapps-vendortesting/ExtensionBundles/Microsoft.Azure.Functions.ExtensionBundle.Workflows/1.28.2/Microsoft.Azure.Functions.ExtensionBundle.Workflows.1.28.2_any-any.zip](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcdnforlogicappsv2.blob.core.windows.net%2Flogicapps-vendortesting%2FExtensionBundles%2FMicrosoft.Azure.Functions.ExtensionBundle.Workflows%2F1.28.2%2FMicrosoft.Azure.Functions.ExtensionBundle.Workflows.1.28.2_any-any.zip&data=05%7C01%7Cv-govubilish%40microsoft.com%7Cfc64597f1d414eebeea708db738238bc%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C638230775484905108%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=X3COeqQjGm9dM2gxR11Ry2QC76q7bwedZA1PmclN1jk%3D&reserved=0)

AB#24361195

**Comment by AbodeSaafan** [comment]
This is expected, the azure function action is an "in-app" action and not an Azure Managed Connection action

**Comment by vgouth** [comment]
@AbodeSaafan . Thanks for the update. Do we need to update the test case for new designer going forward?

![image](https://github.com/Azure/LogicAppsUX/assets/113517384/fb6253fa-66bf-4f8b-9771-a2481c7259c2)

**Comment by ccastrotrejo** [comment]
Hi, in previous designer we used to filter the actions that their capabilities require an azure connection. As for the new designer that implementation is missing, therefore, there is no need to update the test case. We will fix this bug

**Comment by v-prakumar** [comment]
@ccastrotrejo Issue is fixed in VSIX 2.43.0
